py/h2o_wave/server.py

```diff
@@ -112,6 +112,7 @@ def __init__(
             client_state: Expando,
             args: Expando,
             events: Expando,
+            headers: Dict[str, str]
```

Comment:
`http.Header` is 
```go
type Header map[string][]string
```
https://pkg.go.dev/net/http#Header

this is equivalent to `map[string]string`

`self._headers` below is equivalent to `map[string]map[string]string`

I think that the annotation is mismatched.

WDYT?

Am I missing something?

Reply:
Good catch! I forgot that HTTP spec allows multiple values for a single key hence the proper type shall be `Dict[str, List[str]]`.

> self._headers below is equivalent to map[string]map[string]string

Actually it's `map[string][]string`. Since we retrieve the headers from JSON payload via `body.get('headers', None)` so we go from `map[string]map[string][]string` to `map[string][]string`